buildSrc/build.gradle

```diff
@@ -21,7 +21,7 @@ import org.gradle.internal.jvm.Jvm
 import org.gradle.util.GradleVersion
 
 plugins {
-  id 'java-gradle-plugin'
+  id 'groovy-gradle-plugin'
```

Comment:
What's the difference here? Do we still need to expliclty apply the `groovy` plugin?

Reply:
A leftover from figuring out why the test kit integ tests didnt work. should be revertible. will check